src/managers/views/iframe.js

```diff
@@ -146,20 +146,23 @@ class IframeView {
 				// apply the layout function to the contents
 				this.settings.layout.format(this.contents);
 
-				// Expand the iframe to the full size of the content
-				this.expand();
-
-				// Listen for events that require an expansion of the iframe
-				this.addListeners();
+        setTimeout((function() {
```

Comment:
If this is going to be asynchronous, it would be better to wrap it in a promise. 
I think it might also work better with `requestAnimationFrame` as that will trigger on the next repaint.
```
this.addListeners();
return new Promise((resolve, reject)) => {
	requestAnimationFrame(() => {
		this.expand();
		this.emit("rendered", this.section);
		resolve();
	});
});
```